### ISSUE TITLE
SFTP server always prepends home directory to requests

### ISSUE THREAD

**Issue by an0t8** [PowerShell/Win32-OpenSSH#126]
Any time that I try to access a folder or file via sftp, it sticks the home directory in front of the request.  I cannot change the home directory in sshd_config, Chroot doesn't seem to work.

Example:  I connect and the remote directory is C:/Users/username
I try to change to C:/MyDir: >cd C:/Mydir

The server tries to go to C:/Users/username/C:/Mydir

WinSCP even tries to use fully qualified paths within the home directory, so that clicking on a directory "SubDir" tries to access C:/Users/username/C:/Users/username/SubDir

Have I missed a configuration step or is this a bug?

**Comment by quamrulmina** [comment]
Please use the latest build. What you describe works and should work in the latest builds. cd C:/Mydir will work in sftp clients. Forward slash directory path like Unix is used in directory naming. Chroot feature does not work in Windows. In Windows sftp server, home directories are set to the user's actual home directories ( like c:/users/username or c:/users/suername.domainname ) . You should also undo your changes in sshd_config for home directory if cd c:/Mydir still does not work in your environment after using the latest build.

**Comment by an0t8** [comment]
Ok, the 2/25 build fixed the issue.

I still can't get public key authentication working.  I saw that public key authentication isn't supported with domain logons yet, but I created a local account just for sftp.  I tried generating a key pair with ssh-keygen and through puttygen.  Put the public key in the C:\Users\user\.ssh\autorized_keys file, and access is always denied.

Also, with the new build on Windows 7, the ssh-lsa install script did not work correclty.  "[Microsoft.Win32.RegestryKey] doesn't contain a method named 'OpenBaseKey'"  I installed ssh-lsa into my registry manually.
